test error on py 3.6, 3.7 with pytest 5

i'm getting the following test error on py 3.6; I thought it was my branch - but I retested it with a fresh install of the current master.  this passes on py2.7; i did not test other py3.x versions.

```
_______________________________________________________________ tests/test_clean.py::test_only_text_is_cleaned _______________________________________________________________

    def test_only_text_is_cleaned():
        some_text = 'text'
        some_type = int
        no_type = None
    
        assert clean(some_text) == some_text
    
        with pytest.raises(TypeError) as e:
            clean(some_type)
>       assert "argument cannot be of 'type' type" in str(e)
E       assert "argument cannot be of 'type' type" in '<ExceptionInfo TypeError tblen=3>'
E        +  where '<ExceptionInfo TypeError tblen=3>' = str(<ExceptionInfo TypeError tblen=3>)

tests/test_clean.py:27: AssertionError
==================================================================== 1 failed, 311 passed in 5.98 seconds ====================================================================
ERROR: InvocationError: '/Users/jvanasco/webserver/os-projects-others-forks/bleach/.tox/py36/bin/pytest -v'
```

--- a/.gitignore
+++ b/.gitignore
@@ -15,3 +15,5 @@ docs/_build/
 *~
 *.swp
 __pycache__
+venv/
+.idea/

--- a/requirements-dev.txt
+++ b/requirements-dev.txt
@@ -1,7 +1,7 @@
 -e .
 
 # Requirements to run the test suite
-pytest<5
+pytest
 pytest-wholenodeid
 flake8
 tox

--- a/tests/test_clean.py
+++ b/tests/test_clean.py
@@ -24,11 +24,11 @@ def test_only_text_is_cleaned():
 
     with pytest.raises(TypeError) as e:
         clean(some_type)
-    assert "argument cannot be of 'type' type" in str(e)
+    assert "argument cannot be of 'type' type" in str(e.value)
 
     with pytest.raises(TypeError) as e:
         clean(no_type)
-    assert "NoneType" in str(e)
+    assert "NoneType" in str(e.value)
 
 
 def test_empty():

